### ISSUE TITLE
Page behind a session leads to error on Lighthouse report

### ISSUE THREAD

**Issue by pocesar** [harlan-zw/unlighthouse#92]
### Describe the bug

Even though the payload.html are properly created in the `reports` folder (no error there, the page loaded properly), all `lighthouse.json` files contains this:

```json
  "fetchTime": "2023-05-12T00:53:53.678Z",
  "gatherMode": "navigation",
  "runtimeError": {
    "code": "ERRORED_DOCUMENT_REQUEST",
    "message": "Lighthouse was unable to reliably load the page you requested. Make sure you are testing the correct URL and that the server is properly responding to all requests. (Status code: 401)"
  },
  "runWarnings": [
    "Lighthouse was unable to reliably load the page you requested. Make sure you are testing the correct URL and that the server is properly responding to all requests. (Status code: 401)"
  ],
```

### System / Nuxt Info

```Shell
System:
    OS: Linux 5.15 Debian GNU/Linux 11 (bullseye) 11 (bullseye)
    CPU: (6) x64 AMD Ryzen 7 5800X3D 8-Core Processor
    Memory: 14.22 GB / 15.62 GB
    Container: Yes
    Shell: 5.8 - /usr/bin/zsh
  Binaries:
    Node: 18.16.0
    Yarn: 1.22.19
    npm: 9.5.1
```

**Comment by harlan-zw** [comment]
Hi @pocesar, thanks for the issue.

I think this may be related to the site you're scanning. You'll need to provide it for me to investigate further.

**Comment by harlan-zw** [comment]
Hm interesting, I've pushed up a change that may fix it in 0.7.2 (sets the cookies in the hook).
